### PR TITLE
added setMe (update/put)

### PR DESCRIPTION
**Pull request by doktordirk**

### CODE REVIEW

File: src/authService.js [hunk 1]
```diff
@@ -20,6 +20,15 @@ export class AuthService {
     return this.rest.find(this.auth.getProfileUrl());
   }
 
+  setMe(profile) {
+    return this.rest.update(this.auth.getProfileUrl(), profile)
+      .then(response => {
+        return response;
```
Comment (RWOverdijk) [inline src/authService.js]: This `.then()` can be removed. It's returning what it gets.

File: src/authService.js [hunk 2]
```diff
@@ -20,6 +20,15 @@ export class AuthService {
...
+      }).catch(err => {
```
Comment (RWOverdijk) [inline src/authService.js]: This catch should be removed, it's up to the caller to handle exceptions. (I realize it's also in login, would you mind taking that one out, too?)

**Comment by RWOverdijk** [comment]
Good and simple, just the way I like it! :)

**Comment by doktordirk** [comment]
thought those where strange, but i though safest would be to copy them. so now i went the other direction and kicked them from link/unlink as well

**Comment by RWOverdijk** [comment]
Boss!

**Pull request merged**